lib/options/dcu.robot

```diff
@@ -147,3 +147,26 @@ Make Sure That Flash Locks Are Disabled
     Get Flashrom From Cloud
     ${out_flashrom}=    Execute Command In Terminal    flashrom -p internal
     Should Not Contain    ${out_flashrom}    read-only
+
+Login To Windows
+    Power On
+    Boot System Or From Connected Disk    ${OS_WINDOWS}
+
+Boot System Or From Connected Disk
+    [Arguments]    ${os}
+    ${os_boot_id}=    Set Variable    ${EMPTY}
+
+    Login To Linux
+    Switch To Root User
+
+    ${boot_entries}=    SSHLibrary.Execute Command    efibootmgr
+    @{lines}=    Split To Lines    ${boot_entries}
+    FOR    ${line}    IN    @{lines}
+        IF    '${os}' in '${line}'
+            ${os_boot_id}=    Set Variable    ${line}
+        END
+    END
+    ${os_boot_id}=    Split String    ${line}    separator=    max_split=1
+    ${id}=    Get Substring    ${os_boot_id}    4    7
```

Comment:
Don't we need to check if the OS boot entry actually exists here?

Reply:
added: 8e0c7704d287b909dc808fb3a87f8f7ab0fabe90

Reply:
Thanks.